src/components/moj-banner/template.njk

```diff
@@ -0,0 +1,19 @@
+<div class="moj-banner{% if params.type == 'success' %} moj-banner--success{% elif params.type == 'warning' %} moj-banner--warning{% else %} {{- ' ' + params.classes if params.classes}}{% endif %}">
```

Comment:
Can you append an attributes object to the main `div` container for consistency?

```
{%- for attribute, value in params.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}
```